solr/solrj/src/java/org/apache/solr/client/solrj/impl/LBSolrClient.java

```diff
@@ -480,14 +480,35 @@ private static boolean isTimeExceeded(long timeAllowedNano, long timeOutTime) {
     return timeAllowedNano > 0 && System.nanoTime() > timeOutTime;
   }
 
+  private NamedList<Object> doMakeRequest(Endpoint endpoint, SolrRequest<?> solrRequest)
+      throws SolrServerException, IOException {
+    final var solrClient = getClient(endpoint);
+    return doMakeRequest(solrClient, endpoint.getBaseUrl(), endpoint.getCore(), solrRequest);
+  }
+
+  // TODO This special casing can be removed if either: (1) SOLR-16367 is completed, or (2)
+  // LBHttp2SolrClient.getClient() is modified to return a client already pointed at the correct URL
+  private NamedList<Object> doMakeRequest(
```

Comment:
This method name feels awkward, wouldn't a method named requestWithBaseUrl be clearer and consistent?

Reply:
I ended up going with `doRequest` based on your "use overload" comments on L613 and L721 below